go/signedexchange/verifier.go

```diff
@@ -154,66 +154,65 @@ func (e *Exchange) Verify(verificationTime time.Time, certFetcher CertFetcher, l
 		// TODO: Implement Step 6 and 7 (certificate verification).
 
 		// Step 8: "Return "valid"."
-		return true
+		return true, decodedPayload
 	}
-	return false
+	return false, nil
 }
 
 // verifySignature verifies single signature, as described in
 // https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#signature-validity.
```

Comment:
We'd want comments about the returning values.

Reply:
Done.